.github/workflows/build-lint-test.yml

```
@@ -61,6 +61,24 @@ jobs:
             - name: TypeCheck
               run: yarn type:check
 
+    build-example-docker:
+        runs-on: ubuntu-20.04
+        strategy:
+            matrix:
+                image:
+                    - { dir: eSealing, dockerfile: Dockerfile }
+                    - { dir: voting, dockerfile: Dockerfile }
+                    - { dir: wCCD, dockerfile: Dockerfile, extraArguments: '--build-arg NETWORK="mainnet"' }
```

Since we host both `mainnet/testnet` version of the dapp. I suggest we just test building the image for both environmental variables in the CI pipeline as well.

```suggestion
                    - { dir: wCCD, dockerfile: Dockerfile, extraArguments: '--build-arg NETWORK="mainnet"' }
                    - { dir: wCCD, dockerfile: Dockerfile, extraArguments: '--build-arg NETWORK="testnet"' }
```

---

To me it seems excessive to test the build with both versions of the value.